api/package.json

```diff
@@ -19,7 +19,7 @@
     "AMOUNT": 10000000000000,
     "DECIMALS": 12,
     "SS58_PREFIX": 42,
-    "PROVIDER": "ws://localhost:9933",
+    "PROVIDER": "ws://ws.t0rn.io:1933",
```

Comment:
If we go thru public DNS let's use `wss://`

Reply:
True, true. Fixed